xbmc/platform/linux/AppParamParserWebOS.h

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright (C) 2005-2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "AppParamParserLinux.h"
+
+class CAppParamParserWebOS : public CAppParamParserLinux
+{
+public:
+  CAppParamParserWebOS();
+  ~CAppParamParserWebOS();
+
+protected:
+  void ParseArg(const std::string &arg) override;
+
+private:
+  int m_nArgs;
```

Comment:
use default initializer instead of init in ctor definition. `int m_nArgs;` => `int m_nArgs{0};`